content/en/docs/v3.5/install.md

```diff
@@ -86,14 +86,122 @@ $ brew update
 $ brew install etcd
 ```
 
-3. Verify install 
+3. Verify install
 ```sh
 $ etcd --version
 ```
 
 ## Linux
 
-TBD
+Although it is possible to install etcd through major distribution's official repositories and package managers, the versions published are significantly outdated. So, installing this way is strongly discouraged.
+
+The following package managers are community maintained and have the most up-to-date version:
+
+### Homebrew on Linux
+
+[Homebrew can run on Linux], and can provide recent software versions.
+
+* Prerequisites
+  * Update Homebrew:
+    ```sh
+    $ brew update
+    ```
+* Procedure
+  * Install using `brew`:
+    ```sh
+    $ brew install etcd
+    ```
+* Result
+  * Verify installation by getting the version:
+    ```sh
+    $ etcd --version
+    etcd Version: {{< psubstr git_version_tag 1 >}}
+    ...
+    ```
+
+### Alpine Linux
+
+Alpine Linux provides two separate packages: `etcd` and `etcd-ctl`. They are in the testing repository, so you must enable it before installing.
+
+* Prerequisites
+  * Enable testing repository:
+    ```sh
+    $ echo 'https://dl-cdn.alpinelinux.org/alpine/edge/testing/' | sudo tee -a /etc/apk/repositories
+    # or if you're running as root:
+    $ echo 'https://dl-cdn.alpinelinux.org/alpine/edge/testing/' >> /etc/apk/repositories
+    ```
+* Procedure
+  * Install using apk:
+    ```sh
+    $ sudo apk add etcd etcd-ctl
+    # or if you're running as root:
+    $ apk add etcd etcd-ctl
+    ```
+* Result
+  * Verify installation by getting the version:
+    ```sh
+    $ etcd --version
+    etcd Version: {{< psubstr git_version_tag 1 >}}
+    ...
+    ```
+
+### ArchLinux (AUR)
+
+The [Arch User Repository (AUR)] keeps an up-to-date version.
+
+* Prerequisites
+  * Ensure you have installed the `base-devel` package:
+    ```sh
+    $ sudo pacman -S base-devel
+    ```
+  * Fetch the most recent snapshot by going to https://aur.archlinux.org/packages/etcd
+  * Extract the downloaded tar.gz file:
+    ```sh
+    $ tar -zxf etcd.tar.gz
+    ```
+  * Change the working directory:
+    ```sh
+    $ cd etcd
+    ```
+* Procedure
+  * Run `makepkg`:
+    ```sh
+    $ makepkg -s
+    ```
+  * Install the generated pkg file:
+    ```sh
+    $ sudo pacman -U etcd-*.pkg.tar.zst
+    ```
+* Result
+  * Verify installation by getting the version:
+    ```sh
+    $ etcd --version
+    etcd Version: {{< psubstr git_version_tag 1 >}}
+    ...
+    ```
+
+### ALT Linux Sisyphus
```

Comment:
I had never heard of this distro before.  I think we have two decisions to make:

1) Should we go down to the individual distribution level, or just promote brew as our linux install of choice?

2) If we go to the individual distro level, should we stick to a subset of more well known distributions?

Curious what folks think.

Reply:
My two cents:

1. I'd advise promoting brew as install of choice, as using multiple package managers is usually not considered a best practice
2. A compromise might be well known distributions + individual distro where it requires extra steps (eg. `$ sudo distro-package-manager install etcd` is not enough)

Reply:
If we're not going to support unofficial sources (i.e., ArchLinux AUR, Homebrew, etc.), I feel like we should just comment that the suggested way of installing etcd on Linux is through the released binaries.

Reply:
Since it's clear that installing via released binaries is the best due to the statement in line 96, why not focus on popular distros like Debian; Ubuntu, and Fedora; RHEL that are widely used in Enterprises?

See how they were used on the [Kubernetes documentation](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/upgrading-linux-nodes/#upgrade-kubeadm).

Reply:
As far as I know, Kubernetes can do that because it owns and publishes packages to `pkg.k8s.io`—unfortunately, all of the official repositories have out-of-date etcd packages.

Reply:
@network-charles for RHEL instructions see #789

Reply:
@ivanvc not every repository, see [Repology](https://repology.org/project/etcd/versions)

Reply:
Oh, thanks, @d1nuc0m.

> @network-charles for RHEL instructions see #789

Reply:
In the interest of getting this pull request merged let's focus on Homebrew as our package manager for Mac and Linux and remove other distro specific repositories which are generally outdated or problematic.

The Homebrew etcd package is consistently maintained https://formulae.brew.sh/formula/etcd and has been over a long period of time.

Reply:
Done.